website_form_recaptcha/static/src/js/field_recaptcha.js

```diff
@@ -6,7 +6,6 @@ odoo.define('website_form_recaptcha.recaptcha', function(require){
   "use strict";
   
   var snippet_animation = require('web_editor.snippets.animation');
-  var $ = require("$");
   
   snippet_animation.registry.form_builder_send = snippet_animation.registry.form_builder_send.extend({
```

Comment:
I'll make this change in #267, which will then rebase here when it's merged - would you mind also dropping a review there seeing as you already technically reviewed the code?